faster_sam/dependencies/helpers.py

```diff
@@ -0,0 +1,15 @@
+def build_messages_attributes(attributes: dict[str, str]) -> dict[str, dict[str, str]]:
+    if not attributes:
+        return {}
+
```

Comment:
Acho que não precisa dessa validação.

Se algo diferente do que está definido no argumento for passado um erro deve ser gerado.

E caso seja passado um dicionário vazio, um dicionário vazio é retornado.